community/kernel/src/main/java/org/neo4j/kernel/recovery/Recovery.java

```diff
@@ -82,11 +99,27 @@ public void init() throws Throwable
         }
 
         monitor.recoveryRequired( recoveryFromPosition );
+        spi.startRecovery();
+
+        // Backwards for neo store only
+        long checkpointTxId = TransactionIdStore.BASE_TX_ID;
+        try (   TransactionCursor transactionsToRecover = spi.getTransactionsInReverseOrder( recoveryFromPosition );
+                RecoveryApplier recoveryVisitor = spi.getRecoveryApplier( REVERSE_RECOVERY ); )
```

Comment:
Three spaces here may be a little over the top.

Reply:
Just wanted to align both those. Easier on the eye imho.

Reply:
Auto format will remove the extra spaces from both lines. At least for me 😄  doesn't work for you?